app/clients/sms/aws_pinpoint.py

```diff
@@ -0,0 +1,61 @@
+from time import monotonic
+
+import boto3
+import phonenumbers
+from botocore.exceptions import ClientError
+
+from app.clients.sms import SmsClient
+
+
+class AwsPinpointClient(SmsClient):
+    """
+    AWS Pinpoint SMS client
+    """
+
+    def init_app(self, current_app, statsd_client, *args, **kwargs):
+        self._client = boto3.client("pinpoint-sms-voice-v2", region_name="ca-central-1")
+        super(AwsPinpointClient, self).__init__(*args, **kwargs)
+        # super(SmsClient, self).__init__(*args, **kwargs)
+        self.current_app = current_app
+        self.name = "pinpoint"
+        self.statsd_client = statsd_client
+
+    def get_name(self):
+        return self.name
+
+    def send_sms(self, to, content, reference, multi=True, sender=None):
+        pool_id = self.current_app.config["AWS_PINPOINT_POOL_ID"]
+        messageType = "TRANSACTIONAL"
+        matched = False
+
+        for match in phonenumbers.PhoneNumberMatcher(to, "US"):
+            matched = True
+            to = phonenumbers.format_number(match.number, phonenumbers.PhoneNumberFormat.E164)
+            destinationNumber = to
+
+            try:
+                start_time = monotonic()
+                response = self._client.send_text_message(
+                    DestinationPhoneNumber=destinationNumber,
+                    OriginationIdentity=pool_id,
+                    MessageBody=content,
+                    MessageType=messageType,
+                    ConfigurationSetName=self.current_app.config["AWS_PINPOINT_CONFIGURATION_SET_NAME"],
+                )
+            except ClientError as e:
+                self.statsd_client.incr("clients.pinpoint.error")
+                raise Exception(e)
+            except Exception as e:
```

Comment:
I think we can combine these two if the logic is exactly the same.

```suggestion
            except (ClientError, Exception) as e:
```

There might be syntactic sugar to avoid the tuple and rather use the `|` operator nowadays, not sure..

Reply:
🤔 a `ClientError` is an `Exception` so I think we can just get rid of the `ClientError` bit alltogether!